radio/src/translations/en.h

```diff
@@ -156,11 +156,11 @@
 #endif
 
 #define TR_SF_SCREENSHOT               "Screenshot"
-#define TR_SF_RACING_MODE              "RacingMode"
+#define TR_SF_RACING_MODE              "Racing Mode"
 #define TR_SF_DISABLE_TOUCH            "No Touch"
 #define TR_SF_RESERVE                  "[reserve]"
 
-#define TR_VFSWFUNC                    TR_SF_SAFETY,"Trainer","Inst. Trim","Reset","Set",TR_ADJUST_GVAR,"Volume","SetFailsafe","RangeCheck","ModuleBind",TR_SOUND,TR_PLAY_TRACK,TR_PLAY_VALUE,TR_SF_RESERVE,TR_SF_PLAY_SCRIPT,TR_SF_RESERVE,TR_SF_BG_MUSIC,TR_VVARIO,TR_HAPTIC,TR_SDCLOGS,"Backlight",TR_SF_SCREENSHOT,TR_SF_RACING_MODE,TR_SF_DISABLE_TOUCH TR_SF_TEST
```

Comment:
@gagarinlg A thought occurred to me when changing this on the different languages... do you think it would be better to change blocks like this so that *all* of the, for example, TR_VFSWFUNC texts are from defines, rather than inline? i.e. Breaking out to TR_SF_VOLUME, TR_SF_BACKLIGHT, etc? This would effectively mean across all the translations that the actual TR_VFSWFUNC implementation is exactly the same. 

Meaning we know what each block does, even if a different language ;)

Reply:
The knowledge what it means is probably not very useful, most of the time.
This idea would allow us to reduce the number of duplicate strings in the translations and force consistent translations of used right, so I like it.